### PR TITLE
0-intro_Allocene

### PR DESCRIPTION
**Pull request by Allocene**
add README.md

### CODE REVIEW

**Comment by Allocene** [comment]
homework

File: package-lock.json [hunk 1]
```diff
@@ -1,84 +1,134 @@
 {
```
Comment (vladyslav-yermolin-moc) [inline package-lock.json:1]: Hi, @Allocene, please remove this file from your pr

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (Allocene) [reply package-lock.json]: Hello, yep, i`m done.

Comment (vladyslav-yermolin-moc) [reply package-lock.json]: Thanx, but you've removed the whole file. We need to keep the version of the file from the main branch https://github.com/MastersAcademy/fe-course-basic-2023/blob/main/package-lock.json

Comment (Allocene) [reply package-lock.json:1]: i'm not really sure what i need to do (i restored p.json)

Comment (vladyslav-yermolin-moc) [reply package-lock.json:1]: BTW in the materials you can find the answear https://dangitgit.com/ua#%D1%81%D0%BA%D0%B0%D1%81%D1%83%D0%B2%D0%B0%D1%82%D0%B8-%D0%BA%D0%BE%D0%BC%D1%96%D1%82

Comment (Allocene) [reply package-lock.json:1]: how i can find answer if i dont know what the problem? [Чорт забирай, мені треба скасувати коміт який я зробив блін 5 комітів тому]
i removed file -- "please remove this file from your pr"
i add this file again -- "Thanx, but you've removed the whole file. We need to keep the version of the file from the main branch"
i'm new with this stuff, come on

Comment (vladyslav-yermolin-moc) [reply package-lock.json:1]: Ok, I've asked you to remove the fire from your pr, not from the whole repo.
To do so, you need to change the file content back to the state int the repo.
1. You can create another branch (e.g. Allocene:0-intro_Allocene-clean) from the main and commit only your changes with readme. Without package.loc
2. You can revert commits, where you changed the package.lock
3. You can copy-paste content of[ the original file](https://github.com/MastersAcademy/fe-course-basic-2023/blob/main/package-lock.json) to your file and push to this branch

Comment (Allocene) [reply package-lock.json:1]: Ok, so I understand we have some issues with package-lock.json, but I haven't changed it, so I don't really understand what the problem is.
Is there a way where I can just delete the entire repository and download it again?

Comment (vladyslav-yermolin-moc) [reply package-lock.json:1]: Ou, it's a dirty hack (but I also have done it in the past)))
First of all - you've created your own clone repository (https://github.com/Allocene/fe-course-basic-2023/tree/0-intro_Allocene), and the flow is - clone our https://github.com/MastersAcademy/fe-course-basic-2023/tree/main
So yes, you can delete it and clone the local copy of our.
After you can work init, as I've added you as a contributor.
Start from the scratch.

Comment (vladyslav-yermolin-moc) [reply package-lock.json:1]: And after check the files, which you like to push - this is very useful habit.

**Pull request closed without merging**